website/docs/d/yundun_bastionhost_instances.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "alicloud"
```

Comment:
please add a subcategory in the line 2: `subcategory: "Cloud Bastionhost"`, like other docs.